globalconfig.xcconfig

```diff
@@ -14,4 +14,6 @@ OTHER_CFLAGS[sdk=iphonesimulator9.*] = $(MOBILE_CENTER_WARNING_FLAGS) -fembed-bi
 OTHER_CFLAGS[sdk=iphoneos10.*] = $(MOBILE_CENTER_WARNING_FLAGS) -fembed-bitcode
 OTHER_CFLAGS[sdk=iphonesimulator10.*] = $(MOBILE_CENTER_WARNING_FLAGS) -fembed-bitcode-marker
 
+APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
This means, when set to YES, that we guarantee to use App extension API only. Which is true while integrated on app extensions.